packages/geolocator/tizen/src/geolocator_tizen_plugin.cc

```diff
@@ -11,28 +11,122 @@
 #include <flutter/plugin_registrar.h>
 #include <flutter/standard_method_codec.h>
 
-#include <memory>
-#include <string>
```

Comment:
These standard libs are used in this file. Please don't remove the includes. We don't rely on transitive inclusion.

https://google.github.io/styleguide/cppguide.html#Include_What_You_Use

Reply:
I have modified the header include codes by referring to the guide. please check again